.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
```

Comment:
Do we want to set up that this only happens on non-draft PRs?

Reply:
Ah yes, I guess that would be a good idea, I'll add that in my next PR once I've added in all the other submodules